opds_import.py

```diff
@@ -266,7 +266,7 @@ def import_from_feed(self, feed, even_if_no_author=False,
             except Exception, e:
                 identifier, ignore = Identifier.parse_urn(self._db, key)
                 data_source = DataSource.lookup(self._db, self.data_source_name)
-                failure = CoverageFailure(identifier, traceback.format_exc(), data_source=data_source, transient=False)
+                failure = CoverageFailure(identifier, traceback.format_exc(), data_source=data_source, transient=True)
```

Comment:
But this means we'll be retrying imports with problems that aren't fixed yet. It seems like there are really three kinds of failure:
- Something that will go away by itself if we try again later
- Something that will go away once we do something to fix it
- Something that will never go away